stubs/Markdown/markdown/treeprocessors.pyi

```diff
@@ -1,20 +1,26 @@
 from _typeshed import Incomplete
+from re import Pattern
 from typing import Any
+from xml.etree.ElementTree import Element
 
 from . import util
 
 def build_treeprocessors(md, **kwargs): ...
 def isString(s): ...
 
 class Treeprocessor(util.Processor):
-    def run(self, root) -> Any | None: ...
+    def run(self, root: Element) -> Element | None: ...
```

Comment:
The docstring refers to this method as taking an `ElementTree` object, and `xml.etree.ElementTree.ElementTree` is a different class to `xml.etree.ElementTree.Element`. Is the docstring incorrect?

Reply:
Good question, but yes, it’s really an `Element`, not an `ElementTree`: note [this `.getroot()` call](https://github.com/Python-Markdown/markdown/blob/3.4.1/markdown/core.py#L264).

Reply:
Thanks! I hate typing XML 🙃